.gitignore

```diff
@@ -1,4 +1,10 @@
 /.env
 /vendor/
+/wp-content/backup-db/
+/wp-content/backups/
+/wp-content/blogs.dir/
+/wp-content/cache/
+/wp-content/upgrade/
```

Comment:
Do these entries actually appear in your install, or are they "just in case"?

Reply:
I taken them from the default gitignore for wordpress, maybe we can remove the backup folder but the others can avoid the conflicts with other plugin and other stuff that happen locally.

Reply:
Ok - let's skip all of these for now, and add them if we actually need them.  No reason to put things in here that we don't actually need for this site.

- `blogs.dir` was removed in WP 3.5 so we will never see it.
- `backups` and `cache` are not in the default ClassicPress `.gitignore` file.
- `upgrade` is not needed either since we will handle upgrades here on GitHub.